sdk/maps/azure-maps-search/README.md

```diff
@@ -104,20 +83,26 @@ set MAPS_SAMPLES_CONNECTION_STRING="<connection string of your Maps service>"
 
 pip install azure-maps-search
 
-python samples\search_client_sample.py
-python samples\search_client_sample_async.py
+python samples\search.py
 ```
 
-## Troubleshooting
+### Troubleshooting
```

Comment:
Troubleshooting, Next Steps, and Contributing should keep using `##` instead of `###` :)